### PR TITLE
As/2527 wbtc withdraw fix

### PR DESCRIPTION
**Pull request by adamskrodzki**
# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

File: blockchain/utils.ts [hunk 1]
```diff
@@ -23,7 +23,11 @@ export function funcSigTopic(v: string): string {
 
 export function amountToWei(amount: BigNumber, token: string): BigNumber {
   const precision = getToken(token).precision
-  return amount.times(new BigNumber(10).pow(precision))
+  const retValue = amount.times(new BigNumber(10).pow(precision));
+  console.log(`amountToWei = ${retValue}`);
+  const realRetValue = new BigNumber(retValue.toFixed(0,BigNumber.ROUND_DOWN));
```
Comment (adamskrodzki) [inline blockchain/utils.ts:28]: cut all decimals after a comma since wei should be integer number

Comment (peculiarity) [reply blockchain/utils.ts:28]: Clean the implementation. remove all the loggin and simplify it

File: features/manageVault/manageVaultTransactions.ts [hunk 2]
```diff
@@ -279,8 +279,8 @@ export function manageVaultWithdrawAndPayback(
   txHelpers$
     .pipe(
       first(),
-      switchMap(({ sendWithGasEstimation }) =>
-        sendWithGasEstimation(withdrawAndPayback, {
+      switchMap(({ sendWithGasEstimation }) => {
+        return sendWithGasEstimation(withdrawAndPayback, {
```
Comment (peculiarity) [inline features/manageVault/manageVaultTransactions.ts:283]: Do we need this change ?

File: blockchain/utils.ts [hunk 3]
```diff
@@ -23,7 +23,11 @@ export function funcSigTopic(v: string): string {
 
 export function amountToWei(amount: BigNumber, token: string): BigNumber {
   const precision = getToken(token).precision
-  return amount.times(new BigNumber(10).pow(precision))
+  const retValue = amount.times(new BigNumber(10).pow(precision));
```
Comment (peculiarity) [inline blockchain/utils.ts:26]: Having this 10 hardcoded ....Doesn't look good. Where did you get this one from ?

**Pull request closed without merging**